### DISCUSSION TITLE
extreme pitch moment

### DISCUSSION THREAD

**Discussion by Uisliu** [OpenFAST/openfast#2067] [Q&A]
Hi all,
I have been recently running OpenFAST simulations on the IEA 15 MW RWT semi model and have been recovering this warning message

![image](https://github.com/OpenFAST/openfast/assets/71447919/309383af-e1aa-4ad7-9236-84745bfddd2a)

I don't recall changing anything In the settings that may have caused this message to occur. I also cant see other warnings that happen due to it being spammed multiple times per second

If anyone has any input as to where this error might be coming from it would be greatly appreciated

**Comment by jjonkman** [comment]
Dear @Uisliu,

I'm not sure why you are receiving this error, but perhaps it is useful for you to know that this warning message is triggered via MoorDyn.

Best regards,

**Reply by Uisliu** [reply]
Hi @jjonkman,
I was able to fix the problem. but admittedly it was only fixed by redownloading the model

thank you for the input : )
